### DISCUSSION TITLE
jenkins node connection error : Agent JVM has terminated. Exit code=1 Launch failed - cleaning up connection [SSH] Connection closed

### DISCUSSION THREAD

**Discussion by Zeeshanhyder01** [learndevopsonline/learndevopsonline#1621] [Q&A] [answered]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/119431024/9f67939a-4976-4da2-8268-5483e739d322)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/119431024/6ba97413-99b3-4952-8e17-aa296ce3193c)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/119431024/11e6cc89-1047-4b6e-b58c-4f0624f9c648)

**Comment by sreedharm07** [comment]
Hi @Zeeshanhyder01 
can you give ping jenkins password i tried admin123 its not logging in

**Reply by Zeeshanhyder01** [reply]
jenkins
user:admin
password : admin@123

**Reply by Zeeshanhyder01** [reply]
@sreedharm07 did u able to logged in ?

**Reply by sreedharm07** [reply]
yes, am checking

**Reply by sreedharm07** [reply]
Hi @Zeeshanhyder01 
i tried but am not getting why this isuse is.
may be raghu will look into this discussion .

**Reply by Zeeshanhyder01** [reply]
@r-devops sir please take a look sir

**Reply by Zeeshanhyder01** [reply]
@sreedharm07 Thank You

**Reply by Zeeshanhyder01** [reply, marked as answer]
Hi @sreedharm07  @r-devops  *Error Rectified*

Error occurs due to javPath is not setup properly. I was able to fix it by specifying javaPath to correct java version.

**Comment by Zeeshanhyder01** [comment]


**Reply by sreedharm07** [reply]
issue is resolved right ?

**Reply by Zeeshanhyder01** [reply]
@sreedharm07 yes ,solved

**Reply by sreedharm07** [reply]
ok

**Reply by Zeeshanhyder01** [reply]
Hi @sreedharm07  could you please check the code ; there is error coming only we run the job with jenkins, normally its fine working , during jenkins job its dns not updating with ip adresses . please check once  and let me know the problem, Thank you.



![image](https://github.com/learndevopsonline/learndevopsonline/assets/119431024/6117627a-32f5-4f82-a5b7-1dc09f6479a9)

**Reply by sreedharm07** [reply]
if it is only getting error in jenkins 
please clean workspace of jenkins

**Reply by sreedharm07** [reply]
also it shows like permission denied 
check iam role and other permissions which required 

and may be you are from old batch  using centos 7.  and am not much aware of centos 7 commands and procedure

**Reply by Zeeshanhyder01** [reply]
Hi, cleaned the workspace and build again having the same error : create-ec2.sh: line 25: /tmp/record.json: Permission denied
